qiskit_sphinx_theme/pytorch_base/sidebar.html

```diff
@@ -18,7 +18,7 @@
             <div class="sidebar-l1-expandable">Previous Releases</div>
             <div class="sidebar-subheadings">
                 {% for version in version_list | sort(reverse=True) %}
-                    <div class="sidebar-l2"><a href="/documentation/stable/{{ version }}/index.html">{{ version }}</a></div>
+                    <div class="sidebar-l2"><a href="#" onclick="redirectToVersion( {{ version }} )">{{ version }}</a></div>
```

Comment:
Following @eddybrando suggestion, I think nofollow might help here with SEO.

```suggestion
                    <div class="sidebar-l2"><a href="#" onclick="redirectToVersion( {{ version }} )" rel=”nofollow”>{{ version }}</a></div>
```

Reply:
Good suggestion! Thanks.